core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client}
+  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client.  This protocol falls back to Java Web Start Agent Protocol/2 (insecure) when it can't create a secure connection.  This protocol is not recommended.  Use Java Web Start Agent Protocol/4 instead.}
```

Comment:
I think the best way would be to just add change declaration to `${%description}` in Jelly and then add the `description.properties` file with `description=$THE_EXISTING_TEXT`. And then field names in localizations can be just replaced